rovercode/app.py

```diff
@@ -0,0 +1,181 @@
+"""Rovercode app."""
+import websocket
```

Comment:
It would be nice to order the imports
* Standard library
* External library
* Project

Reply:
I'm surprised prospector didn't check that. Fixing.